src/cart/README.env

```diff
@@ -2,14 +2,13 @@ This file lists the environment variables used in CaRT.
 
  . D_PROVIDER (Deprecated: CRT_PHY_ADDR_STR)
    It determines which mercury NA plugin to be used:
-   1: set it as "ofi+verbs;ofi_rxm" to use OFI verbs;ofi_rxm provider
-   2: set it as "ofi+psm2" to use OFI psm2 provider
-   3: set it as "ofi+gni" to use OFI gni provider
-   4: set it as "sm" to use SM plugin which only works within single node
-   5: set it as "ofi+tcp;ofi_rxm" to use OFI tcp;ofi_rxm provider.
-   6: set it as "ofi+sockets" to use OFI sockets provider
+   - set it as "ofi+verbs;ofi_rxm" to use OFI verbs;ofi_rxm provider
+   - set it as "ofi+gni" to use OFI gni provider
```

Comment:
we should remove ofi+gni references too, i dont even know what gni provider is :)

Reply:
Yes sure I was not trying to rewrite the readme as part of that PR but if I go that route we should make it match the crt type list yeah or refer to the doc if there is any doc.

Reply:
I will leave the ofi+gni reference for now. I'd prefer to focus that PR on psm2 and not also remove the gni one from the cart code, we can do that later.